docs/hardware.rst

```diff
@@ -272,39 +272,13 @@ Specific Hardware Recommendations
 Application and Monitor Servers
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-We currently recommend the Gigabyte BRIX, Intel NUC, or Mac Mini for
-SecureDrop servers.
+We currently recommend the Intel NUC or Mac Mini for SecureDrop servers.
```

Comment:
If we're going to continue to recommend the Mac Mini, we need to be explicit about either year (2014) or model numbers, and probably make a point of saying the 2018 ones won't work.

Reply:
As of your changes in 2732588c68c8ad2ff0091ff47866b5f0b1a2af3f we at least have a warning to that effect. I think we should keep the docs around (given the significant install base on Mic Mini), but we shouldn't say that "we _currently_ recommend .. the Mac Mini". I'll take a stab at refactoring this section accordingly in a follow-up commit in this PR.

Reply:
This is now done; @zenmonkeykstop, could you take another look?